backend/store/stacktraces.go

```diff
@@ -170,14 +170,20 @@ func (store *Store) EnhanceTraceWithGitHub(ctx context.Context, trace *privateMo
 	return &newStackTraceInput, nil
 }
 
-func (store *Store) GitHubEnhancedStackTrace(ctx context.Context, stackTrace []*privateModel.ErrorTrace, workspace *model.Workspace, project *model.Project, errorObj *model.ErrorObject) ([]*privateModel.ErrorTrace, error) {
+func (store *Store) GitHubEnhancedStackTrace(ctx context.Context, stackTrace []*privateModel.ErrorTrace, workspace *model.Workspace, project *model.Project, errorObj *model.ErrorObject, testService *model.Service) ([]*privateModel.ErrorTrace, error) {
```

Comment:
i'm not a huge fan of the `test` prefix for the argument (same thing with the resolver). it's confusing since `test` could mean a bunch of things - could we change this to `validateService`?